### PR TITLE
systemd-hwdb: add page

### PR DESCRIPTION
**Pull request by varadrane7**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-hwdb.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Update the binary hardware database:
+
+`systemd-hwdb update`
+
+- Query the hardware database and print the result for a specific modalias:
+
+`systemd-hwdb query {{modalias}}`
+
+- Update the binary hardware database, but return a non-zero exit value on any parsing error:
```
Comment (acuteenvy) [inline pages/linux/systemd-hwdb.md:14]:
```suggestion
- Update the binary hardware database, returning a non-zero exit value on any parsing error:
```

File: pages/linux/systemd-hwdb.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`systemd-hwdb --strict update`
+
+- Update the binary hardware database, and generate it in /usr/lib/udev instead of /etc/udev:
```
Comment (acuteenvy) [inline pages/linux/systemd-hwdb.md:18]:
```suggestion
- Update the binary hardware database in `/usr/lib/udev`:
```

File: pages/linux/systemd-hwdb.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# systemd-hwdb
+
+> Hardware database management tool.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-hwdb.html>.
+
+- Update the binary hardware database:
```
Comment (acuteenvy) [inline pages/linux/systemd-hwdb.md:6]:
```suggestion
- Update the binary hardware database in `/etc/udev`:
```

File: pages/linux/systemd-hwdb.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Update the binary hardware database, but return a non-zero exit value on any parsing error:
+
+`systemd-hwdb --strict update`
+
+- Update the binary hardware database, and generate it in /usr/lib/udev instead of /etc/udev:
+
+`systemd-hwdb --usr update`
+
+- Update the binary hardware database, specifying an alternate root path in the filesystem:
```
Comment (acuteenvy) [inline pages/linux/systemd-hwdb.md:22]:
```suggestion
- Update the binary hardware database in the specified root path:
```

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**